dockerfiles/testsuite/start_test.sh

```diff
@@ -11,6 +11,7 @@
 #  e2etest_ballotelection: run ballot test
 #  e2etest_lifecyclelection: run lifecycle test
 #  e2etest_cspelection: run csp test
+#  e2etest_dynamicensustelection: run dynamic census test
```

Comment:
why the `t`?
i think `dynamicensuselection` would be more consistent with the rest

Reply:
yep, it was a typo

Reply:
comment still has the typo. (the others are fixed, nice)